bittensor/_neuron/text/advanced_server/run.py

```diff
@@ -126,9 +126,13 @@ def priority(pubkey:str, request_type:bittensor.proto.RequestType, inputs_x) ->
                     torch inputs to be forward processed.
                 request_type ( bittensor.proto.RequestType, `required`):
                     the request type ('FORWARD' or 'BACKWARD').
-        """        
-        uid = metagraph.hotkeys.index(pubkey)
-        priority = metagraph.S[uid].item()/ sys.getsizeof(inputs_x)
+        """
+        try:        
+            uid = metagraph.hotkeys.index(pubkey)
+            priority = metagraph.S[uid].item()/ sys.getsizeof(inputs_x)
+        
+        except:
+            return 0
```

Comment:
what did we say about `except` nothing? 🙃

Reply:
haha oops, its supposed to be returning zero priority. Let me fix that XD